api/app.yaml

```diff
@@ -6,4 +6,4 @@ entrypoint: gunicorn -b ":$PORT" data_explorer.__main__:app
 env_variables:
   PATH_PREFIX: /api
   ELASTICSEARCH_URL: http://ELASTICSEARCH_URL:9200/
-  DATASET_CONFIG_DIR: /app/dataset_config
+  DATASET_CONFIG_DIR: /app/dataset_config/platinum_genomes
```

Comment:
This deployment config should be unset rather than pointing to test data by default.

Reply:
Why -- everywhere else in this and data-explorer-indexer repo, we use a test dataset by default.

Also, I like having a sample string here. Otherwise, it's not clear that it should start with "/app/dataset_config". And it's not clear if this should be "/app/dataset_config" or "/app/dataset_config/platinum_genomes".